tests/Integration/Security/Verifier/JwtTokenVerifierTest.php

```diff
@@ -60,4 +62,11 @@ public function testUnsuccessfulVerification(): void
     {
         self::assertFalse($this->subject->isValid($this->createMock(Token::class)));
     }
+
+
+    public function testUnsuccessfulToken(): void
+    {
+        $token = (new ReflectionClass(Plain::class))->newInstanceWithoutConstructor();
```

Comment:
Create classes without constructor is a bad practice. Better to use some mock data.